sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/State.java

```diff
@@ -13,25 +13,25 @@ class State {
 
     private final Instant nextRefreshCheck;
 
-    private final String key;
+    private final String originEndpoint;
 
```

Comment:
Maybe we should make this State class more general, unaware of the actual state it holds.

Reply:
State hold refresh information outside of the refresh scope.
* Etag info of watch keys
* `Instant` of the next refresh
* Origin Endpoint of the store the state is for
* Count of the number of failed refreshAttempts of the store since the last success.
* Time between refresh attempts, used to calculate `Instant` value.